src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,38 @@
 
 #include "t06_homework.h"
 #include <iostream>
-
+#include <set>
 using namespace std;
 
 int t06_homework() {
+int D,F = 0;
+cin >> D;
+string S1,S2;
+set<string> MSET;
+    for (int L = 0; L < D; L++) {
+    cin >> S1;
+    MSET.insert(S1);}
+ while (cin >> S2) {
+  bool CSET;
+  CSET =MSET.count(S2);
+  int CT = 1,CF = 0;
 
+    if (CSET)
+    {   CT = 0;}
+    else {char Conv;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'Conv' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4512320)